docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to Phonebook 
+
+## How can I contribute?
+### Reporting Bugs
+You can contribute by reporting bugs. For that there are some rules:
+- Before submitting please check that the same issue was not already submitted.
+- Use the [Bug-Report-Template](https://github.com/T-Systems-MMS/phonebook/issues/new?assignees=&labels=bug&template=bug_report.md&title=) and answer the questions.
+
+### Suggesting new Features
+If you have new ideas or have something that you really need in Phonebook you can submit a new feature request. For that there are some rules:
+- Before submitting please check that the same feature request was not already submitted.
+- Use the [Feature-Request-Template](https://github.com/T-Systems-MMS/phonebook/issues/new?assignees=&labels=feature&template=feature_request.md&title=) and answer the questions.
+
+### Your First Code Contribution
+Unsure where to begin contributing to Phonebook? You can start by looking for the ["good-first-issue"-label](https://github.com/T-Systems-MMS/phonebook/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+If you have questions about the issue or do not know where to start you can just ask under the issue.
+
+### Open Pull Requests
+If you want to open a pull request please use our [template](will be added soon).
+
+### Reviewing a pull request
+
+## What should I know before I get started?
```

Comment:
maybe this should be first in the document.

First, think about what you want to do and then think about the rest.